### DISCUSSION TITLE
For Urgent Support Needs

### DISCUSSION THREAD

**Discussion by jzongker** [ChurchApps/ChurchAppsSupport#332] [General]
Please use this forum for any non-urgent support needed.  If your need is urgent or you wish to get in touch for other reasons, you can reach us via:

**Email:**
 [support@livecs.org](mailto:support@livecs.org)

**Phone:**
 +1 (918) 994-2638

**FB Messenger:**
 https://m.me/livecsolutions

**Comment by WFMCCAdmin** [comment]
https://github.com/orgs/ChurchApps/discussions/393

**Comment by martin-grannis** [comment]
I'm trying to drag multiple png's from folder into an empty show and have them all as foregrounds. They seem to always go as backgrounds. Is there an easy way?

**Reply by vassbo** [reply]
Drag it onto a slide in the "Edit" page instead of in the "Show" page.
Or right click the image in the drawer, click "Edit", set the type to "Foreground".

**Comment by fabiocastillo76-bit** [comment]
<img width="1440" height="900" alt="Captura de pantalla 2026-05-23 a la(s) 6 39 27 p m" src="https://github.com/user-attachments/assets/e0a2bd66-700d-4cb9-8182-4d249cc701fd" />
<img width="1440" height="900" alt="Captura de pantalla 2026-05-23 a la(s) 6 40 29 p m" src="https://github.com/user-attachments/assets/f00639c1-00dc-4029-8778-c03868c2de34" />
Hello from La Patagonia, Chile. I was trying to work with groups and calendars but I can't see some group members. I decided to add me intentionally as a leader in each groups but it doesn´t work. What I'm doing wrong? God bless you.
